README.md

```diff
@@ -322,15 +271,9 @@ cd youki
 just youki-dev # or youki-release
 ```
 
-# Community
-
-We also have an active [Discord](https://discord.gg/h7R3HgWUct) if you'd like to come and chat with us.
```

Comment:
Maybe we should add the discord link in the docs contributing section, I don't think it is there right now.